nixos/modules/services/networking/pihole-ftl-setup-script.nix

```
@@ -22,8 +22,10 @@ in
   pihole="${lib.getExe pihole}"
   jq="${lib.getExe pkgs.jq}"
 
+  ${lib.getExe pkgs.curl} "${lib.strings.escapeShellArg cfg.macvendorURL}" -o "${cfg.settings.files.macvendor}"
```

```suggestion
  ${lib.getExe pkgs.curl} --retry 3 --retry-delay 5 "${lib.strings.escapeShellArg cfg.macvendorURL}" -o "${cfg.settings.files.macvendor}"
```
Maybe this is a good idea?